P0009/P0009.bs

```diff
@@ -1904,24 +1908,34 @@ Table �: Accessor policy requirements
   <td>`A::offset_policy`</td>
   <td></td>
   <td>Accessor policy for accessing a pointer returned by `a.offset(p,i)`. 
-      <br/> *Requires:* `A::offset_policy` meets the requirements of an accessor policy in Table �.
+      <br/>
+      *Requires:*
+      + `A::offset_policy` meets the requirements of an accessor policy in Table �.
+      + `is_convertible_v<A, A::offset_policy>` is `true` and `A::offset_policy(a)` is valid.
```

Comment:
Why the is_convertible? Where do we need that?

Reply:
If you have a stateful `AccessorPolicy`, there needs to be a way to propagate that information to the `offset_policy`.  This is how `Allocator`s work, and I think most people would expect `subspan` to work this way also. How else would you propose propagating the `AccessorPolicy` to the `subspan`?